sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc

```diff
@@ -208,11 +200,10 @@ namespace sycl::ext::oneapi::experimental {
 `root_group`, `group` or `sub_group`.
 
 `is_user_constructed_group<T>::value` is `std::true_type` if `T` is one of:
-`ballot_group`, `fixed_size_group`, `tangle_group`, or `opportunisic_group`.
+`fragment` or `chunk`.
 
 Additionally, the `is_group<T>::value` trait from SYCL 2020 is `std::true_type`
```

Comment:
```suggestion
Additionally, the `is_group<T>::value` trait from the core SYCL specification is `std::true_type`
```

Future proofs for new versions of the core SYCL specification.  Same comment for the other spec.

Reply:
Fixed in b4461cb4d2b3c6c0f8c424e2d5fdd0184de070ec.